src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -0,0 +1,512 @@
+	"FrameDuration": [10, 10, 10, 10, 10, 10, 20, 20, 20, 20, 20, 20, 60, 60, 60, 60, 60, 60, 120, 120, 120, 120, 120, 120, 120, 120, 300, 300, 300, 300, 300, 300, 300, 300, 300, 300, 300, 300, 300, 300, 300, 300, 300, 300, 300],
+	"AcquisitionMode": "list mode",
+	"ImageDecayCorrected": true,
+	"ImageDecayCorrectionTime": 0,
+	"ReconMethodName": "3D-OSEM-PSF",
+	"ReconMethodParameterLabels": ["subsets","iterations"],
+	"ReconMethodParameterUnit": ["none","none"],
+	"ReconMethodParameterValues": [16,10],
+	"ReconFilterType": "none",
+	"AttenuationCorrection": "[137Cs]transmission scan-based"
+}
+```
+
+## Blood recording data
+
+This section includes all data needed to perform blood analysis for PET data.
+The section may be omitted completely if blood measurements of radioactivity were not made.
+If blood measurements were made a `*_blood.json` file is expected that contains
+information about which blood data is available as well as some blood measurement details.
+Furthermore, depending on the information given in the `*_blood.json` file
+we expect `.tsv` files with corresponding .json files that contain the values
+and information regarding plasma samples, metabolite fractions,
+discrete blood samples and/or continuous blood samples from an autosampler.
+All these measurements should be defined according to a single time-scale
+in relation to time zero defined by the PET data (Figure 1).
+Additionally, if blood and metabolite measurements were made
+one should always report radioactivity in plasma and corresponding
+parent fraction measurements, including fractions of radiometabolites.
+All definitions used below are in accordance with Innis et al. 2007 [3].
+All method specific information related to the measurements can be stated
+in the field `description`.
+
+Template:
+
+```Text
+sub-<participant_label>/
+ 	[ses-<session_label>/]
+	pet/
+	sub-<participant_label>[_ses-<session_label>][_task-<task_label>][_acq-<label>][_rec-<label>][_run-<index>]_blood.json
+	sub-<participant_label>[_ses-<session_label>][_task-<task_label>][_acq-<label>][_rec-<label>][_run-<index>]_recording-<label>.tsv
+	sub-<participant_label>[_ses-<session_label>][_task-<task_label>][_acq-<label>][_rec-<label>][_run-<index>]_recording-<label>.json
+
+```
+
+Blood data belongs in the /pet folder along with the corresponding PET data.
+However, the blood data also follows the inheritance principle (https://bids-specification.readthedocs.io/en/stable/02-common-principles.html#the-inheritance-principle) and may be moved to an upper level folder
+if it does not change, for example, with multiple reconstructions.
+The blood data is most often recorded using an autosampler for continuous blood samples,
+or manually for discrete blood samples.
+Therefore, the recording key (`recording-<label>`) has two reserved values:
+`blood_continuous`,
+for continuous whole blood data measurements (2.2.5);
```

Comment:
I am not sure what the "2.2.x" refer to in this section. Are they links to other headings in this document?

Reply:
Hi @Remi-Gau , well spotted, we will fix this. Thanks!